tree/tree/src/TBranch.cxx

```diff
@@ -1544,18 +1547,19 @@ Int_t TBranch::GetBulkEntries(Long64_t entry, TBuffer &user_buf)
 }
 
 ////////////////////////////////////////////////////////////////////////////////
-/// Read as many events as possible into the given buffer, using zero-copy
-/// mechanisms (e.g. no de-seriralization or byte-swaps are performed).
+/// \brief Read as many events as possible into the given buffer without byte swapping.
```

Comment:
Here (and above) "as many as possible" is (and was in the original) inprecise.  We could use a "basket worth of events" (or better worded equivalent).

Reply:
ok, I'd go with "read a basket of events" unless you have a better suggestion